renovate.json5

```diff
@@ -7,10 +7,14 @@
     ":warning: Please do not merge directly :warning:",
     "Follow update procedure in the [README](https://github.com/giantswarm/cloud-provider-vsphere-app?tab=readme-ov-file#how-to-update-the-charts-automatically)."
   ],
-  "ignorePaths": [
```

Comment:
Previously this ignored ALL updates in these directories. Your change will now only ignore Kubernetes manifests. I'm not sure that's what you want is it?